pkg/cache/cache.go

```diff
@@ -0,0 +1,72 @@
+package cache
+
+import (
+	"encoding/json"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// Cache is an interface for a structure susceptible of caching
+type Cache interface {
+	Get(string, interface{})
+	Set(string, interface{})
+	Del(string)
+}
+
+type noCache struct{}
+
+func (*noCache) Get(string, interface{}) {}
+func (*noCache) Set(string, interface{}) {}
+func (*noCache) Del(string)              {}
+
+type jsonCache struct {
+	namespace  string
+	expiration time.Duration
+	client     subRedisInterface
+}
+
+func (c *jsonCache) Get(d string, out interface{}) {
```

Comment:
I'd have expected `Get` to return a bool (or an err) to say if it is a hit or a miss